python/ray/tests/test_chaos.py

```diff
@@ -133,6 +134,31 @@ def get(self):
     # assert_no_system_failure(p, 10000, 10)
 
 
+def test_chaos_defer(monkeypatch, ray_start_cluster):
+    with monkeypatch.context() as m:
+        m.setenv("RAY_grpc_based_resource_broadcast", "true")
+        # defer for 100s
+        m.setenv(
+            "RAY_testing_asio_delay_ms",
+            "NodeResourceInfoGcsService.grpc_client.UpdateResources=100000")
+        m.setenv("RAY_event_stats", "true")
+        cluster = ray_start_cluster
+        cluster.add_node(num_cpus=16, object_store_memory=1e9)
+        cluster.wait_for_nodes()
+        ray.init(address="auto")
+        cluster.add_node(num_cpus=16, num_gpus=1)
+
+        bundle = [{"GPU": 1}, {"CPU": 1}]
+        pg = placement_group(bundle)
+        ray.get(pg.ready())
+
+        @ray.remote
+        def g():
+            return "g"
+
+        print(ray.get([g.options(placement_group=pg).remote()]))
```

Comment:
Does it actually test the functionality?

Reply:
Maybe we need some assert to make sure it takes more than X seconds?

Reply:
I'm still working on the e2e test.